### ISSUE TITLE
Indicators doesn't show

### ISSUE THREAD

**Issue by mastoj** [Jaysce/Spaceman#28]
I've started spaceman, but the indicators in the bar doesn't show for some reason. I can see that the app is running in activity monitor. Since I can't see it there is no way for me to access the preferences. 

How can I debug it?

Running an M2 MacBook Pro with Ventura if that is any help.

**Comment by mastoj** [comment]
I suspect that it is something wrong with the settings since I tried the spaceid app first and I had the same issue. I guess you share some settings/config from the whichspace project, where is that settings stored? Can I edit it manually or restore to the initial settings?

**Comment by Jaysce** [comment]
Since the M2 MacBook Pro has a notch, it may be that you have a lot of menu bar icons in the bar and also have a lot of spaces resulting in it being hidden. You could check to see by closing the other apps.

**Comment by mastoj** [comment]
@Jaysce, I don't think that is the problem. Tried to close some apps, but the spaceman bar icon still doesn't show. It is room for at least 6-7 icons between my notch and the remaining apps, so I don't think that is the problem.

Is there a way I can change settings with having access to the bar icon?

**Comment by Jaysce** [comment]
You could try the following:
1. Quit Spaceman
2. Run `defaults delete dev.jaysce.Spaceman ` in terminal
3. Run Spaceman

This should reset the settings to its default values. Let me know if this helps.
